### PR TITLE
Add basic test validation for 'Network' and 'Connection' object models

### PR DESCRIPTION
**Pull request by rghodgkin**
Summary:
This utilizes pytest to parametrize various positive and negative test scenarios when calling the 'models.load()' function to build Pureport Network and Connection API objects.   See attached file "sample_output_run.txt" for 
[sample_output_run.txt](https://github.com/pureport/pureport-python/files/5220599/sample_output_run.txt)

The following is basic functionality of the test scenarios:
- 'ModelData' by default will return mandatory attributes only
- 'ModelData' will randomize string and integer data with min/max limits from pulled from openapi or randomize ENUM choice
- Attributes manually specified within parametrize will either 'override' existing or 'insert' new
- Running with 'pytest --log-cli-level INFO' will print details of dict data being loaded

### CODE REVIEW

**Comment by rghodgkin** [comment]
Pytest execution within tox fails locally unless the 'PUREPORT_API_KEY' and 'PUREPORT_API_SECRET' inside tox.ini are set to values other than 0.

File: test/models/test_models.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#
+# Copyright (c) 2020, Pureport, Inc.
+# All Rights Reserved
+
+import pytest
+import random
+from pureport import models
+from ..utils import utils
+from .test_helpers import ModelData
+models.make()
```
Comment (privateip) [inline test/models/test_models.py]: This needs to be mocked out and the openapi.json file loaded from `test/openapi.json`

**Review by privateip — CHANGES_REQUESTED** [review]

**Comment by privateip** [comment]
> Pytest execution within tox fails locally unless the 'PUREPORT_API_KEY' and 'PUREPORT_API_SECRET' inside tox.ini are set to values other than 0.

You need to mock the call to`get_api()` and load the spec from `test/openapi.json` instead

File: test/models/test_models.py (hunk 1 above)
Comment (rghodgkin) [reply test/models/test_models.py]: Done.

**Review by privateip — APPROVED** [review]

**Pull request merged**